bundles/org.openhab.binding.dwdpollenflug/src/main/resources/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dwdpollenflug"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+		<label>DWD Pollen count index - Bridge</label>
+		<description>Bridge for accessing pollen count index data of the DWD</description>
+
+		<channel-groups>
+			<channel-group id="updates" typeId="updates"/>
+		</channel-groups>
+
+		<properties>
+			<property name="sender"/>
+			<property name="name"/>
+		</properties>
+
+		<config-description>
+			<parameter name="refresh" type="integer" unit="min" min="15">
+				<default>30</default>
+				<label>Refresh in Minutes</label>
+				<description>Time between two API requests in minutes. Minimum 15 minutes.</description>
+			</parameter>
+		</config-description>
+		
+	</bridge-type>
+	
+	<channel-group-type id="updates">
+		<label>Updates</label>
+		<description>Information about data state</description>
+		<channels>
+			<channel id="refreshed" typeId="update">
+				<label>Bridge refreshed</label>
+			</channel>
+			<channel id="last_update" typeId="update">
+				<label>Last update from DWD</label>
+			</channel>
+			<channel id="next_update" typeId="update">
+				<label>Next update from DWD</label>
+			</channel>
+			<channel id="updated" typeId="updated" />
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="update">
+		<item-type>DateTime</item-type>
+		<label>Update time</label>

```

Comment:
The first letter of every word in a label should be capitalized.
```suggestion
		<label>Update Time</label>

```
Please update elsewhere as appropriate.

Reply:
Is this an OpenHAB convention? Should I although do it like "Refresh In Minutes" or like "Refresh in Minutes". And should it be the same in translation file?

Reply:
Using upper-case letters as first letters in labels is an openHAB convention, yes. This also applies to translations. Please do not use something like "In Minutes". If the unit is set in the description, the UI should properly display the unit.

Reply:
Fixed all labels to upper case and removed in Minutes